cdc/entry/schema_storage.go

```diff
@@ -105,6 +110,17 @@ func WrapTableInfo(info *timodel.TableInfo) *TableInfo {
 				ti.UniqueColumns[ti.Columns[col.Offset].ID] = struct{}{}
 			}
 		}
+		if idx.Primary || idx.Unique {
```

Comment:
Is `primary key` be counted twice, as L104 has increased once.

Reply:
no, if pk is handle, `primary index` will be not included in `ti.Indices`.